packages/use-dataloader/src/__tests__/usePaginatedDataLoader.test.tsx

```diff
@@ -0,0 +1,213 @@
+/* eslint-disable no-console */
+import { act } from '@testing-library/react'
+import { renderHook } from '@testing-library/react-hooks'
+import React from 'react'
+import DataLoaderProvider from '../DataLoaderProvider'
+import { KEY_IS_NOT_STRING_ERROR } from '../constants'
+import {
+  UsePaginatedDataLoaderConfig,
+  UsePaginatedDataLoaderMethodParams,
+  UsePaginatedDataLoaderResult,
+} from '../types'
+import usePaginatedDataLoader from '../usePaginatedDataLoader'
+
+type UseDataLoaderHookProps = {
+  config: UsePaginatedDataLoaderConfig<unknown>
+  key: string
+  method: (params: UsePaginatedDataLoaderMethodParams) => Promise<unknown>
+}
+
+const PROMISE_TIMEOUT = 5
+
+const initialProps = {
```

Comment:
Shouldn't this be typed with `UseDataLoaderHookProps`?

Reply:
We don't care because the renderHooks function is typed so it will throw errors if we dont put the right type inside